### DISCUSSION TITLE
GitHub Copilot Billing via Azure CSP

### DISCUSSION THREAD

**Discussion by mana7331** [community/community#141027] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I understand that when linking an Azure subscription, both GitHub Enterprise Cloud and GitHub Copilot fees are typically charged via Azure. However, I would like to confirm whether it is possible to pay for only GitHub Copilot (GitHub Copilot for Business or GitHub Copilot for Enterprise) through Azure CSP, without being required to include GitHub Enterprise Cloud fees in the same billing.

For reference, I have reviewed the documentation here: [Connecting an Azure subscription to GitHub]
https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/connecting-an-azure-subscription

**Comment by queenofcorgis** [comment, marked as answer]
@mana7331 this depends on your licensing model. 

* Self-serve enterprise account on metered GHE can pay using credit card, PayPal or Azure subscription. Only one payment method is supported for all usage.
* Self-serve enterprise account on volume GHE - must pay for Enterprise licenses using card/PayPal. Metered services can be billed to the card/PayPal on file or customers can connect an Azure subscription for metered usage only.

**Reply by tiagoefreitas** [reply]
can the github enterprise seats also be paid using the azure subscription?

**Comment by mana7331** [comment]
Hi  @queenofcorgis  Thank you for your reply and I understood it!

**Reply by queenofcorgis** [reply]
@mana7331 wonderful, I'm going to mark the answer then and reopen this discussion for better discoverability for our members. Thanks for letting us know this helped ❤️

**Reply by thilkip** [reply]
yes, any additional resources will help, we tried the below link but didnt see the azure subscription 
https://docs.github.com/en/copilot/managing-copilot/managing-github-copilot-in-your-organization/managing-the-copilot-subscription-for-your-organization/about-billing-for-github-copilot-in-your-organization

**Comment by thilkip** [comment]
We have GitHub Organization with free plan and would like to link it to Azure subscription for Copilot Business billing is that possible? if yes, is there a documentation for it?

**Reply by mohammed-mksk** [reply]
Did you manage to resolve this? I have a similar issue

**Reply by thilkip** [reply]
Yes, we did. There is a option on the setting page in the GitHub Org, where you can link them

**Comment by Aloostor** [comment]


**Reply by queenofcorgis** [reply]
Hi @geoarda , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
